.github/workflows/ci-on-comment.yaml

```diff
@@ -0,0 +1,75 @@
+name: Kata Containers CI
+on:
+  issue_comment:
+    types: [created]
+
+jobs:
+  debug:
+    runs-on: ubuntu-latest
+    steps:
+      - name: /test-gha check
+        if: always()
+        id: match-test-gha
+        run: |
+          comment_body=$(
+          cat <<'END_COMMENT'
+          ${{ github.event.comment.body }}
+          END_COMMENT
+          )
+          echo "$comment_body"
+          # Bash regexes can't match on '\n' so use grep.
+          if grep -qE '(^|\n)/test-gha($|\n)' <<< "$comment_body"; then
+            echo "has_command=true" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Check is PR
+        if: always()
+        run: |
+          echo '${{ github.event.issue.pull_request }}'
+
+      - name: Check ok-to-test
+        if: always()
+        run: |
+          echo '${{ contains(github.event.issue.pull_request.labels.*.name, 'ok-to-test') }}'
```

Comment:
I'd drop the need of the `ok-to-test` label.

Reply:
Done